README.md

```diff
@@ -1,6 +1,8 @@
 # vald-client-java
 
 [![Maven Central](https://img.shields.io/maven-central/v/org.vdaas.vald/vald-client-java?style=flat-square)](https://search.maven.org/search?q=g:%22org.vdaas.vald%22%20AND%20a:%22vald-client-java%22)
+[![Vald version](https://img.shields.io/github/release/vdaas/vald.svg?style=flat-square)](https://github.com/vdaas/vald/releases/latest)
+[![java](https://oss.aoapps.com/ao-badges/java-8.svg)](https://docs.oracle.com/en/java/javase/8/)
```

Comment:
Any reason you are choosing Java 8?
Did you tested it with Java 8?
Where is the Java version definition?

I think you should also mention Kotlin's Version.

Reply:
I chose Java 8 because it is LTS of java.
I am sorry that I haven't tested it with Java8, and there is no Java version definition right now, so it is using the java in `ubuntu-latest` to build java client.

About kotlin version, I will find and add a badge for it.

Reply:
Java version of ubuntu-latest is 11

Reply:
PremierSupport for Java 8 ended last year.
I can't believe why you would choose 8.

We've talked before about testing three Java versions: the Latest and two LTS Versions of Java.

The current Latest is 21 and the LTS Latest is 21 and 17.

Reply:
Sorry it was my mistake, Premier support for Java8 is ended last year.
I will test the java client on Java 17 and 21, and update the badge to 17 later.

Reply:
you can use setup-java to manage multiple java versions.
https://github.com/actions/setup-java

Reply:
we decided to create another task to test multiple version of java, I will make use of this action to implement it later.
thank you

Reply:
Please post PR URL after you create New PR.

Reply:
the PR is here 
https://github.com/vdaas/vald-client-java/pull/202